src/System.Threading.Thread/src/System.Threading.Thread.csproj

```diff
@@ -8,6 +8,7 @@
     <!-- Type being defined conflicts with imported type from dependency assembly -->
     <NoWarn>436</NoWarn>
     <ProjectGuid>{06197EED-FF48-43F3-976D-463839D43E8C}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uapaot'">$(DefineConstants);uap</DefineConstants>
```

Comment:
The defines in this project should stay as uapaot. They are temporary ProjectN specific hacks to make things compile; these hacks are not needed for CoreCLR.